src/components/OrderBuySell/index.tsx

```diff
@@ -5,7 +5,7 @@ import Tabs from 'components/common/Tabs/Tabs'
 export interface TabItemType {
   readonly id: number
   title: string
-  content: string
+  content?: string | undefined
```

Comment:
`prop:?`: stuff` is the same as `prop: stuff | undefined`

Reply:
```suggestion
  content?: string | React.ReactNode
```

Should also accept components probably. please test this code doesn't break it

Reply:
> `prop:?: string` is the same as `prop: string | undefined`

almost
`{props?: string} ===roughly==== {props: string} | {}`
if it's undefined you have to pass `undefined`, [see here](https://www.typescriptlang.org/play?target=99#code/JYWwDg9gTgLgBAJQKYEMDG8BmUIjgcilQ3wChTgA7GJKTdJOANQmABMAFHMAZzgG9ScOBwQB5DgH4AXHB4woVAOakAvqQD0GuDAAWSHoxgB3CDoCeYA3BRE4AaypsUO-XJQgkAGjgAjAK7w6DD+KAA2YeZwlBAwpDCWjCzsXBC8AExwALwCohKy8oqUSqpwAD4C6qRoEJTycADCuGDJbLLIwQB0AGINADytqbwAfNlwABRg3DwAlNmjfWDD-FNpPJ15HKp9GkvVtfVN4K3p7cQwPf2D0+mjOZPTc1kLS-z4m-hwVHCrvHAAZP8ftMNuItjs9vs6vAqPIji1WGwAIJjPrw1oaYZQ+qwmDoxEAIVR+PYIjBWQARDw2BTMdiYdCSWx0iicmjmic6TVoV9GRzEekiWymZlNpTqbSsRRqLR6GhGABVShsJCYKhITjTARCMn5OQKZTlOD+ZWq9VsNTkemNZpKlVqygas5dXp9O1mx2atZ3Ca-WbzOCLZZ+0ESba7KXcnF88Duh0a1mB+Fx82YuBaOC6FAANyMZjAKB4fE21txydN8bYQqTtor5t1HCy-BN9vNqkxQA)